Userland/Libraries/LibJS/Runtime/ArrayConstructor.cpp

```diff
@@ -139,7 +139,7 @@ JS_DEFINE_NATIVE_FUNCTION(ArrayConstructor::from)
                 return {};
 
             if (!next) {
-                array_object.put(vm.names.length, Value(k));
+                array_object.set(vm.names.length, Value(k), true);
```

Comment:
I honestly have no idea what `true` is for, can we have an enum class instead?

Reply:
no, the specification requires that parameter to be a boolean

Reply:
The spec also requires the "hint" argument to `get_iterator` to be a string, I don't see your point.

Reply:
that's true, it's indeed a 1:1 spec translation, which doesn't always make sense - not opposed to change it to a yes/no enum. `Object::SetThrow::Yes/No`?

Reply:
i mean, thats not true, the hint in getIterator is a type, not a string, but see the part in parenthesis

Reply:
> Object::SetThrow::Yes/No

That's what I meant by enum class to begin with

Reply:
Let's worry about this later, this PR is all copy-the-spec anyway, changing one bool isn't gonna suddenly make anything better or worse